### PR TITLE
MAINT-38133 : fix wiki page with images export pdf issue

### PR DESCRIPTION
**Pull request by mzorai**
This PR should fix the issue of exporting wiki page with images to pdf files

### CODE REVIEW

File: wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java [hunk 1]
```diff
@@ -120,5 +128,29 @@ private File createPDFFile(String title, String content) throws IOException {
...
+      for(Element element : elements){
+        if(element.tagName().equals("img") && !element.attr("src").isEmpty()){
+          String src = element.attr("src");
+          String imgName = src.substring(src.lastIndexOf("/") + 1);
+          Attachment imageAttachment = wikiService.getAttachmentOfPageByName(imgName, page, true);
+          if(imageAttachment == null) {
+            Page parentPage = wikiService.getParentPageOf(page);
+            imageAttachment = wikiService.getAttachmentOfPageByName(imgName, parentPage, true);
+          }
+          if(imageAttachment != null && imageAttachment.getMimeType().startsWith("image/")){
```
Comment (hakermi) [inline wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java:145]:
```suggestion
          if(imageAttachment != null && imageAttachment.getMimeType() != null && imageAttachment.getMimeType().startsWith("image/")){
```
something missing here should be in PR 261

**Review by hakermi — CHANGES_REQUESTED** [review]

File: wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java (hunk 1 above)
Comment (hakermi) [inline wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java]: re-add PR 261 again after PR 260

**Review by hakermi — CHANGES_REQUESTED** [review]

File: wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java (hunk 1 above)
Comment (ahamdi) [reply wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java]: @hakermi coul dyou help and update the PR ?

Comment (hakermi) [reply wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java]: Done

**Review by hakermi — APPROVED** [review]

**Review by ahamdi — APPROVED** [review]

**Pull request merged**